### DISCUSSION TITLE
[Suggestions] AI should give greater priority to the resources available on the adventure map

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#8801] [General]
While playing games and watching let's play videos, I noticed that fheroes2's AI was neglecting the capture of resource stacks, among other things.
The importance of resources in Heroes 2, and fheroes2, is paramount.
The AI should collect these resources as quickly as possible, if not for itself, in order to prevent other players, and particularly the human player, from collecting them.

In the original game, the AI gives the human player as little opportunity as possible to take advantage of all the resources at disposal, be they resource generators, resource stacks, treasure chests and so on.

**Comment by ihhub** [comment]
Will the AI be better by doing this? It will slow down the AI expansion on a map.

**Reply by LeHerosInconnu** [reply]
Hello @ihhub,

If it's possible for AI to be better without worrying about resources, that's fine, I am okay with that. :)
